models/src/main/protobuf/CasperMessage.proto

```diff
@@ -241,21 +241,10 @@ message RChainState {
   int64         blockNumber  = 3;
 }
 
-message PaymentDeploy {
-  Par    code      = 1; // payment code (short leash)
-  bytes  codeHash  = 2;
-  bytes  userPk    = 3; // user public key
-  int64  timestamp = 4;
-  uint64 amount    = 5;
-  uint64 phloRate  = 6;
-  uint64 nonce     = 7;
-  bytes  sig       = 8; // signature over (codeHash, pk, timestamp, amount, rate, nonce)
-}
-
 message Deploy {
-  Par           term        = 1; //rholang term to deploy
-  DeployData    raw         = 2;
-  PaymentDeploy paymentCode = 3;
+  Par           term    = 1; //rholang term to deploy
+  DeployData    raw     = 2;
+  Par           payment = 3;
```

Comment:
@goral09 Just so I'm clear, this is a field that holds the code that Alice uses to pay Bob for evaluating `term`?